website/content/ja/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: TAG Environmental Sustainability
+toc_hide: true
+---
+
+<div class="row mt-5 mb-3">
+    <div class="col-lg-6">
+        <div class="lead">
+        TAG Environmental Sustainabilityは、クラウドネイティブアプリケーションのデリバリに関する構築、パッケージング、デプロイ、管理、運用を含むプロジェクトとイニシアチブをサポートします。
+        </div>
+    </div>
+    <div class="col-lg-6">
+        <img src="/images/tag-environmental-sustainability_color.svg" alt="Tag Environmental Sustainability logo" style="max-width: 300px;">
+    </div>
+</div>
+
+<p class="mt-5 mb-5"><img src="/images/tag-env-sustainability-header.webp" alt="TAG Environmental Sustainability"></p>
+
+このテクニカルアドバイザリーグループの目的は、クラウドネイティブテクノロジーにおける環境の持続可能性イニシアチブの開発やサポート、そして評価するのに役立つ提唱をします。このTAGは、クラウドネイティブツールを通じてサービスプロバイダーが消費量やカーボンフットプリントを削減するための価値と可能なインセンティブを特定します。
+
+<!-- cSpell:ignore Linktree -->
+TAG Environmental Sustainabilityの最も重要なリンクは、Linktreeページから利用できます: [cncfenvtag](https://linktr.ee/cncfenvtag).
+
+- [GitHubリポジトリ](https://github.com/cncf/tag-env-sustainability)
+- [TAG Charter](https://github.com/cncf/tag-env-sustainability/blob/main/charter.md)
+- [イベント](https://tag-env-sustainability.cncf.io/events/)
+- Slack チャンネル:
+  - [#tag-environmental-sustainability](https://cloud-native.slack.com/archives/C03F270PDU6)
+  - [#tag-env-wg-comms](https://cloud-native.slack.com/archives/C068XUD9AEA)
+  - [#tag-env-wg-green-reviews](https://cloud-native.slack.com/archives/C060EDHN431)
+  - [Invite yourself to the CNCF Slack](https://slack.cncf.io/)
+- ソーシャルメディアアカウント:
+  - [LinkedIn](https://www.linkedin.com/company/cncf-tag-environmental-sustainability)
+  - [Twitter/X](https://twitter.com/CNCFEnvTAG)
+- [メーリングリスト](https://lists.cncf.io/g/cncf-tag-env-sustainability/topics)
+- [サーベイ](https://github.com/cncf/tag-env-sustainability/tree/main/artifacts/surveys)
+
+## Meetings
+
+毎月第1、第3水曜日の16:00 UTC ([convert to your local
+time](https://dateful.com/convert/utc?t=16)).
```

Comment:
I see that the quality check build failed with this link being invalid and when I try to navigate to this link it doesn't seem to be working for me (getting connection timeout). Can you please check this link? Thanks!

Reply:
Thank you for your check!
dateful.com is not working now(Pages translated into other languages also point to the same link).

I think we need to use another site, or the page we are translating to should state the time in that language.

Reply:
It seems the links working again. Maybe rerun the test?